api/src/pcapi/serialization/utils.py

```diff
@@ -9,13 +9,16 @@
 
 
 def to_camel(string: str) -> str:
+    # used to define root level lists, see https://docs.pydantic.dev/1.10/usage/models/#custom-root-types
+    if string == pydantic_v1.utils.ROOT_KEY:
+        return pydantic_v1.utils.ROOT_KEY
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise** ‏